lib/usd/ui/layerEditor/saveLayersDialog.cpp

```diff
@@ -501,8 +495,8 @@ void SaveLayersDialog::onSaveAll()
                 auto qFileName = row->absolutePath();
                 auto sFileName = qFileName.toStdString();
 
-                auto newLayer = UsdMayaSerialization::saveAnonymousLayer(
-                    sdfLayer, sFileName, parentLayer, row->usdFormatTag().toStdString());
+                auto newLayer
+                    = UsdMayaSerialization::saveAnonymousLayer(sdfLayer, sFileName, parentLayer);
```

Comment:
row->usdFormatTag() was still tracking the option, but without any UI to change it now it is stuck at binary.  This is really the only change needed to fix the defect, everything else is removing code that shouldn't be active now that we have the new optionVar + menu item.